packages/wdio-allure-reporter/src/index.js

```diff
@@ -101,12 +101,14 @@ class AllureReporter extends WDIOReporter {
         const currentTest = this.allure.getCurrentTest()
 
         if (!this.isMultiremote) {
-            const { browserName, deviceName } = this.capabilities
-            const targetName = browserName || deviceName || cid
-            const browserstackVersion = this.capabilities.os_version || this.capabilities.osVersion
-            const version = browserstackVersion || this.capabilities.version || this.capabilities.platformVersion || ''
-            const paramName = deviceName ? 'device' : 'browser'
-            const paramValue = version ? `${targetName}-${version}` : targetName
+            // eslint-disable-next-line camelcase
+            const { browserName, device, deviceName, version, platformVersion, os_version, osVersion } = this.capabilities
+            const targetName = browserName || device || deviceName || cid
+            // eslint-disable-next-line camelcase
```

Comment:
Can't you can do something like this instead?:

```
// eslint-disable camelcase
... some code
// eslint-enable camelcase
```

Reply:
Maybe be better then


```
// eslint-disable-next-line camelcase  
const {..., os_version: OsVersion, ...}
```

Reply:
Sure can. I will update this today.

Reply:
@baruchvlz do you think I should still block this out?

@BorisOsipov are you saying to use the `disable-next-line` feature or to use a block?

Reply:
@mikesalvia I meant you can rename variable on object destructing with name without underscore e.g. `os_version: OsVersionName` and disable eslint for one line only :)

Reply:
Ahhh perfect! 👍 Great idea! 😄

Reply:
DONE! 😄